core/templates/dev/head/pages/collection_editor/CollectionEditorNavbarDirective.js

```diff
@@ -39,7 +39,8 @@ oppia.directive('collectionEditorNavbar', [function() {
         $scope.collectionRights = (
           CollectionEditorStateService.getCollectionRights());
 
-        $scope.isPageLoading = CollectionEditorStateService.isLoadingCollection;
+        $scope.isCollectionLoading = (
```

Comment:
For consistency, please pick one of: ``isCollectionLoading`` and ``isLoadingCollection`` and use everywhere.